src/Esfa.Vacancy.Infrastructure/Services/CachedGetAllApprenticeMinimumWagesService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Esfa.Vacancy.Application.Interfaces;
+using Esfa.Vacancy.Domain.Entities;
+using Esfa.Vacancy.Domain.Interfaces;
+using SFA.DAS.NLog.Logger;
+
+namespace Esfa.Vacancy.Infrastructure.Services
+{
+    public class CachedGetAllApprenticeMinimumWagesService : IGetAllApprenticeMinimumWagesService
+    {
+        private readonly IGetAllApprenticeMinimumWagesService _minimumWagesService;
+        private readonly ILog _logger;
+        private readonly ICacheService _cacheService;
+
+        private const string CacheKey = "VacancyApi.GetAllApprenticeMinimumWages";
+        private const double CacheExpirationHours = 1;
```

Comment:
Who decided this? I think for this is should be higher.

Reply:
I used the same values that we use for getting standard and framework codes.